__tests__/app.test.js

```diff
@@ -176,12 +176,64 @@ describe("GET /api/articles", () => {
         });
       });
   });
-  test("return 404 error when endpoint does not exist", () => {
-    return request(app)
-      .get("/api/not-an-article-route")
-      .expect(404)
-      .then((response) => {
-        expect(response.body.msg).toBe('not found');
-      });
-  });
 });
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('return an array', () => {
```

Comment:
This doesn't need to be an independent test, you're asserting an array in the test below.